scripts/build/ci-build/build-deploy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-_version="1.4.3"
+_version="1.4.5"
```

Comment:
What happened to `1.4.4`? 🤔

Reply:
1.4.4 is on this PR.
Next time we want to update the image, we run the build-deploy.sh and the new version 1.4.5 will be deployed :) 
We just forgot to update the script for 1.4.4 last time!

However, if you too find it annoying, I can remove the script, as its easy to do it manually!

Reply:
Aha, I get it 😄